library-app/src/components/Books/BooksList/ManageBookForm.tsx

```diff
@@ -0,0 +1,219 @@
+              <input
+                type='text'
+                id='title'
+                name='title'
+                defaultValue={formData.Title}
+                onChange={(e) => setFormData((prev) => ({ ...prev, Title: e.target.value }))}
+              />
+            </div>
+            <div className={styles['form-group']}>
+              <label htmlFor='description'>Add Description</label>
+              <textarea
+                id='description'
+                name='description'
+                defaultValue={formData.Description}
+                onChange={(e) => setFormData((prev) => ({ ...prev, Description: e.target.value }))}
+              />
+            </div>
+            <div className={styles['form-group']}>
+              <label htmlFor='isbn'>ISBN</label>
+              <input
+                id='isbn'
+                name='isbn'
+                type='text'
+                defaultValue={formData.Isbn}
+                onChange={(e) => setFormData((prev) => ({ ...prev, Isbn: e.target.value }))}
+              />
+            </div>
+          </div>
+          <div className={styles.right}>
+            <div className={styles['form-group']}>
+              <label htmlFor='quantity'>Quantity</label>
+              <input
+                id='quantity'
+                name='quantity'
+                type='number'
+                defaultValue={formData.Quantity}
+                onChange={(e) => setFormData((prev) => ({ ...prev, Quantity: +e.target.value }))}
+              />
+            </div>
+            <div className={styles['form-group']}>
+              <label htmlFor='publishDate'>Publish Date</label>
+              <input
+                id='publishDate'
+                name='publishDate'
+                type='date'
+                defaultValue={formData.PublishDate}
+                onChange={(e) => setFormData((prev) => ({ ...prev, PublishDate: e.target.value }))}
+              />
+            </div>
+
+            <div className={styles['form-group']}>
+              <label htmlFor='authorIds'>Author(s)</label>
+              <Select
+                className={styles['select-authors-dropdown']}
+                name='authorIds'
+                id='authorIds'
+                options={authors}
+                defaultValue={formData.AuthorIds}
+                getOptionLabel={(option) => `${option.FirstName} ${option.LastName}`}
+                onChange={onChangeAuthors}
+                isMulti
+                getOptionValue={(option: Author) => option.Id.toString()}
+              />
+              {!isOpenForm && (
+                <button onClick={openFormhandler} className={styles['add-btn']}>
+                  Add New Author
+                </button>
+              )}
+            </div>
+          </div>
+        </div>
+        {!isOpenForm && <button className={styles['form-submit-btn']}>Submit Book</button>}
+      </form>
+      {isOpenForm && (
```

Comment:
You use isOpenForm to indicate if author form is active? Then you probably want to call it isAutorFormOpen or something like that to avoid confusion